### PR TITLE
feat(markdownlint): add custom rule to identify unclosed URLs

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Addresses https://github.com/mdn/content/pull/28689#discussion_r1302063973

It's easy to miss the closing bracket in markdown links `[text](/en-US/abc`. The PR adds a custom rule to identify missing closing brackets.

Auto fix is not possible because single replacement string won't work in all cases.

### CODE REVIEW

**Comment by bsmth** [comment]
Thanks @OnkarRuikar - taking a look now 👀

File: files/en-us/web/css/css_basic_user_interface/index.md [hunk 1]
```diff
@@ -49,8 +49,7 @@ To see the code for this basic user interface sample, [view the source on GitHub
 
 ## Guides
 
-- [Using URL values for the `cursor` property](/en-US/docs/Web/CSS/CSS_basic_user_interface
-  Title: CSS basic user interface/Using_URL_values_for_the_cursor_property)
+- [Using URL values for the `cursor` property](/en-US/docs/Web/CSS/CSS_basic_user_interface)
```
Comment (bsmth) [inline files/en-us/web/css/css_basic_user_interface/index.md:52]: This links to the current page and there's a redirect from:

```
/en-US/docs/Web/CSS/CSS_Basic_User_Interface/Using_URL_values_for_the_cursor_property	/en-US/docs/Web/CSS/cursor
```

So this is not a guide as this section says, and we link to cursor already above in `### Properties`, I think we can remove this line or we move it to Related concepts as:

```md
- URL values for the CSS [`cursor`](/en-US/docs/Web/CSS/cursor) property to produce custom cursors.
```

Comment (OnkarRuikar) [reply files/en-us/web/css/css_basic_user_interface/index.md]: The `cursor` property page doesn't explain how to use custom cursors from URLs so I am simply mentioning the property.

**Review by bsmth — APPROVED** [review]
Thanks, leaving a +1

**Pull request merged**